registry/registry.go

```diff
@@ -195,6 +195,51 @@ type Registry struct {
 	views   map[string]*View
 }
 
+// GetViewsAffectedByPath returns all the views in the registry that have visibility
+// into a storage path.
+func (r *Registry) GetViewsAffectedByPath(path string) []*View {
+	var views []*View
+	for _, view := range r.views {
+		var includeView bool
+		for _, rule := range view.rules {
+			if pathChangeAffects(path, rule.originalStorage) {
+				includeView = true
```

Comment:
Why introduce a new variable (`includeView`), when the append could just happen right here?

Reply:
Good point, not sure why I used that (carry over from a previous version maybe?). I've removed it now, thanks